api/apps/articles/test/model/index/persistence.coffee

```diff
@@ -508,6 +508,8 @@ describe 'Article Persistence', ->
         sections: [
           {
             type: 'image_set'
+            layout: 'mini'
+            title: 'The Best Artworks'
```

Comment:
Do you mind adding these to fixtures as well? https://github.com/artsy/positron/blob/master/test/helpers/fixtures.coffee